app/views/course/achievement/achievements/_achievement.html.slim

```diff
@@ -24,9 +24,13 @@
 
   td
     div.btn-group
-      - if can?(:manage, achievement)
+      - if can?(:award, achievement)
```

Comment:
I think there is an `N + 1` query in achievement index ?  for each achievement it will count the conditions.